packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -306,7 +307,8 @@ export interface DDSFuzzSuiteOptions {
 	 */
 	validationStrategy:
 		| { type: "random"; probability: number }
-		| { type: "fixedInterval"; interval: number };
+		| { type: "fixedInterval"; interval: number }
+		| { type: "partialSynchronization"; probability: number; clientProbability: number };
```

Comment:
```suggestion
		// WIP: This validation strategy still currently synchronizes all clients.
		| { type: "partialSynchronization"; probability: number; clientProbability: number };
```

(just in case anyone tries to use it and gets confused while we're in the process of implementing)